client/components/UserPage.js

```diff
@@ -65,10 +64,7 @@ const _UserPage = ({
   );
 };
 
-const mapStateToProps = ({ user, adminUser, userChallenges, individualChallenge }) => {
-  if (user.type === 'admin') {
```

Comment:
Don't delete this.  This is what enables an admin to be able to see other users' userpage info.

Are you sure this is what is causing the problem?

Reply:
@theo333, this is causing the problem for admin Id to show-up, however, I do need admin for admin to see each individual userPage.